src/irmin-unix/jbuild

```diff
@@ -3,5 +3,5 @@
 (library
  ((name        irmin_unix)
   (public_name irmin-unix)
-  (libraries   (irmin irmin-fs git irmin-git irmin-http git-unix irmin-mem
+  (libraries   (irmin irmin-fs checkseum.c git irmin-git irmin-http git-unix irmin-mem
```

Comment:
No, this should be added to the executable. The library are linked against the `cmi` only, the right implementation (C or OCaml) should be chosen at link time.

Reply:
See https://github.com/mirage/irmin/pull/529